Objects/longobject.c

```diff
@@ -6213,11 +6175,7 @@ static PyObject *
 int_bit_length_impl(PyObject *self)
 /*[clinic end generated code: output=fc1977c9353d6a59 input=e4eb7a587e849a32]*/
 {
-    uint64_t nbits = _PyLong_NumBits(self);
-    if (nbits == (uint64_t)-1) {
-        return NULL;
-    }
-    return PyLong_FromUnsignedLongLong(nbits);
+    return PyLong_FromUInt64(_PyLong_NumBits(self));
```

Comment:
You might add `assert(!PyErr_Occurred());` before calling PyLong_FromUInt64().

Reply:
Do you suggest to split this line on two lines and add an assert in-between?

Reply:
I agree with Victor's suggestion.

Reply:
Split into two lines.